04/index.html

```diff
@@ -1,30 +1,114 @@
+    >
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="ie=edge"
+    >
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+
+    <script
+        src="https://kit.fontawesome.com/e6dada8901.js"
+        crossorigin="anonymous"
+    ></script>
+
+    <link
+        href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="./style/reset.css"
+    />
+
+    <link
+        rel="stylesheet"
+        href="./style/global.css"
+    />
+
+    <link
+        rel="stylesheet"
+        media="(max-width:600px)"
+        href="./style/mobile.css"
+    >
+    <link
+        rel="stylesheet"
+        media="(min-width:601px)"
+        href="./style/desktop.css"
+    >
+
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
+    <div class="container">
+        <header>
+            <h1 class="logo">BIOLABS</h1>
+            <nav>
+                <label for="menu-switcher"></label>
+                <input
+                    type="checkbox"
+                    id="menu-switcher"
+                >
                 <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+                    <li class="menu__item">
+                        <a href="#">start</a>
+                    </li>
+                    <li class="menu__item menu__dropdown menu__dropdown--arrow">
+                        <a href="#">oferta</a>
+                        <ul class="dropdown">
+                            <li class="dropdown__item"><a
+                                    href=""
+                                    class="dropdown__link"
+                                >strony internetowe</a></li>
+                            <li class="dropdown__item dropdown__link_2  menu__dropdown--arrow2"><a
+                                    href=""
+                                    class="dropdown__link menu__dropdown"
+                                >pozycjonowanie stron www</a>
+                                <ul class="dropdown__2">
+                                    <li class="dropdown__item"><a
+                                            href=""
+                                            class="dropdown__link"
+                                        >audyt Twojego serwisu</a></li>
+                                    <li class="dropdown__item"><a
+                                            href=""
+                                            class="dropdown__link"
+                                        >monitorowanie efektów</a></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="menu__item">
```

Comment:
Wracając jeszcze do Twojego pytania:
> idealnie jakby rozszerzała się przestrzeń dla hovera i ostatni element menu spadał niżej

"Zwiększenie przestrzeni" dla hovera jest dość prosta. Wystarczy, że np. damy padding dla elemenu, który ma mieć hover lub zwiększymy jego przestrzeć zajmowaną np. - konkretne `height` oraz `width`. Aby element "spadał" to trzeba mu nadpisać `position: absolute` przez `static` :)